virtio-devices/src/iommu.rs

```diff
@@ -282,76 +282,44 @@ unsafe impl ByteValued for VirtioIommuProbeProperty {}
 unsafe impl ByteValued for VirtioIommuProbeResvMem {}
 unsafe impl ByteValued for VirtioIommuFault {}
 
-#[derive(Debug)]
+#[derive(Error, Debug)]
 enum Error {
-    /// Guest gave us bad memory addresses.
+    #[error("Guest gave us bad memory addresses: {0}.")]
```

Comment:
Not sure the punctuation makes sense. Is this consistent with other error descriptions?

Reply:
I looked through our code base and we're inconsistent. Definitely something to follow-up with later.

Reply:
@rbradford Do you think we should keep all error messages without period at the end, given they are (mostly) not complete sentences?